solr/solr-ref-guide/modules/getting-started/pages/introduction.adoc

```diff
@@ -16,28 +16,32 @@
 // specific language governing permissions and limitations
 // under the License.
 
-Solr is a search server built on top of https://lucene.apache.org[Apache Lucene], an open source, Java-based, information retrieval library.
-It is designed to drive powerful document retrieval applications - wherever you need to serve data to users based on their queries, Solr can work for you.
+Apache^TM^ Solr is a search server built on top of https://lucene.apache.org[Apache Lucene^TM^], an open source, Java-based, information retrieval library.
+Solr is designed to drive powerful document retrieval or analytical applications involving unstructured data, semi-structured data or a mix of unstructured and structured data.
+It also has secondary support for limited relational, graph, statistical, data analysis or storage related use cases.
+Since Solr is Apache 2.0 licensed open source software designed for extensibility, it gives you the freedom to adapt or optimize it for almost any commercial or non-commercial use case.
 
-Solr is based on open standards and it is highly extensible.
-Solr queries are simple HTTP request URLs and the response is a structured document: mainly JSON, but it could also be XML, CSV, or other formats.
-This means that a wide variety of clients will be able to use Solr, from other web applications to browser clients, rich client applications, and mobile devices.
-Any platform capable of HTTP can talk to Solr.
-See xref:deployment-guide:client-apis.adoc[] for details on client APIs.
+Solr's xref:query-guide:query-syntax-and-parsers.adoc[query syntax and parsers] offer support for everything from the simplest keyword searching through to complex queries on multiple fields and xref:query-guide:faceting.adoc[faceted] search results.
+xref:query-guide:collapse-and-expand-results.adoc[Collapsing] and xref:query-guide:result-clustering.adoc[clustering] results offer compelling features for e-commerce and storefronts.
+Streaming calculations across large sets of text documents are available via xref:query-guide:streaming-expressions.adoc[streaming expressions].
```

Comment:
can you say why this is cool?  other statements about features give a "why".   What use cases does streaming calcs help?  "Enabling you to break free of the tyranny of the request/response live cycle and process huge amounts of data"???

Reply:
Hmm, good idea, will ponder... slightly more serious suggestions welcome of course ;)